### PR TITLE
[CP-1837] Random crashes while copying files

### PR DESCRIPTION
**Pull request by lukasmas**
Preventing of reading header part which could be to large and cause crash

### CODE REVIEW

File: taglib/mpeg/mpegfile.cpp [hunk 1]
```diff
@@ -502,12 +502,16 @@ void MPEG::File::read(bool readProperties)
...
+  // Prevent of reading ID3v2 header for MPEG files
+  d->ID3v2Location = -1;
+
+    /* OLD CODE
+    d->ID3v2Location = findID3v2();
+    if(d->ID3v2Location >= 0) {
+        d->tag.set(ID3v2Index, new ID3v2::Tag(this, d->ID3v2Location, d->ID3v2FrameFactory));
+        d->ID3v2OriginalSize = ID3v2Tag()->header()->completeTagSize();
+    }
+    */
```
Comment (niepiekm) [inline taglib/mpeg/mpegfile.cpp]: Just remove it. We have git to keep the history of changes.

File: taglib/flac/flacfile.cpp [hunk 2]
```diff
@@ -519,6 +519,16 @@ void FLAC::File::scan()
     const bool isLastBlock = (header[0] & LastBlockFlag) != 0;
     const unsigned int blockLength = header.toUInt(1U, 3U);
 
+    if (blockType >= MetadataBlock::Picture) {
+      nextBlockOffset += blockLength + 4;
```
Comment (niepiekm) [inline taglib/flac/flacfile.cpp]: Please, document where this magic number comes from, and most importantly what is the meaning of it.

**Review by awulkiew — APPROVED** [review]

File: taglib/flac/flacfile.cpp (hunk 2 above)
Comment (lukasmas) [reply taglib/flac/flacfile.cpp]: Done.

File: taglib/mpeg/mpegfile.cpp (hunk 1 above)
Comment (lukasmas) [reply taglib/mpeg/mpegfile.cpp]: Done.

**Pull request merged**